pages.pl/osx/caffeinate.md

```diff
@@ -0,0 +1,16 @@
+# caffeinate
+
+> Nie pozwól aby macOS się uśpił.
+> Więcej informacji: <https://ss64.com/osx/caffeinate.html>.
+
+- Nie usypiaj przez 1 godzinę (3600 sekund):
+
+`caffeinate -u -t {{3600}}`
+
+- Nie usypiaj dopóki komenda nie zostanie zakończona:
```

Comment:
The following code fix can be applied to the original [page](https://github.com/tldr-pages/tldr/blob/main/pages/osx/caffeinate.md):

> Prevent from [s]leeping until a command completes:

Add short option mnemonics when possible.

Reply:
Not really possible here, adding it causes only confusion. 

For example, I could add `u[s]ypiaj`, but it's not unique to this command.  
Using `zo[s]tanie`(~becomes, ~changes state to) also feels weird.